src/reducers/errors.js

```diff
@@ -0,0 +1,38 @@
+import omit from 'lodash.omit';
+
+import * as actionTypes from '../constants/action-types';
+
+const initialState = {};
+
+const queries = (state = initialState, action) => {
+    switch (action.type) {
+        case actionTypes.RESET: {
+            return {};
+        }
+        case actionTypes.MUTATE_START:
+        case actionTypes.REQUEST_START: {
+            const { queryKey } = action;
+
+            return omit(state, queryKey);
```

Comment:
I think it might be better to only clear this on a successful query.  I could imagine a scenario where you have some UI state or even another query config that depends on the error reducer state so you wouldn't want it cleared out straight away.